basket/news/utils.py

```diff
@@ -376,6 +380,25 @@ def get_user_data(
     if get_fxa:
         user["has_fxa"] = bool(user.get("fxa_id"))
 
+    if not user.get("email_id"):
```

Comment:
Oh. I do have one thing. Should we add `and settings.CTMS_ENABLED` here, so that we can push this out whenever we're ready and have it continue to work. Or maybe I'm looking at it wrong and `ctms.get()` will return `None` if CTMS isn't enabled?

Reply:
Looking at it more I think my last question is the deal. Is that what you were thinking?

Reply:
Yes, you have it right. ``ctms.get()`` quickly returns ``None`` when ``settings.CTMS_ENABLED`` is ``False``, due to ``ctms.interface == None``. So, it should be safe to push this code to prod as long as ``settings.CTMS_ENABLED`` is ``False``.